test/test_model.py

```diff
@@ -61,12 +62,25 @@ def config_get(self):
             'qux': True,
         }
 
+    def status_get(self, app):
+        return {
+            'message': '',
+            'status': 'unknown',
+            'status-data': '',
+        }
+
+    def status_set(self, app, status, message=''):
+        self.status_set_calls.append((app, status, message))
+
 
 class TestModel(unittest.TestCase):
     def setUp(self):
         self.backend = TestModelBackend()
         self.model = juju.model.Model('myapp/0', ['db0', 'db1', 'db2'], self.backend)
 
+        juju.model.Unit.status = juju.model.EntityStatus()
+        juju.model.Application.status = juju.model.EntityStatus()
```

Comment:
I take it these are to clear the cached values between tests?  Probably worth adding a comment here.